### PR TITLE
Ch9441/replace deprecated extend usage with styled

### PR DESCRIPTION
**Pull request by philipwindeyer**

### CODE REVIEW

File: packages/components/src/DatePicker/components/CalendarDays/CalendarDays.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
+import styled from 'styled-components';
```
Comment (talbet) [inline packages/components/src/DatePicker/components/CalendarDays/CalendarDays.js]: My OCD always makes me put react at the top of the imports as it is providing the JSX compiler, but this is not required.

Comment (philipwindeyer) [reply packages/components/src/DatePicker/components/CalendarDays/CalendarDays.js]: Haha, I'm with you on that.  Not sure how I missed it.
Updated

Comment (talbet) [reply packages/components/src/DatePicker/components/CalendarDays/CalendarDays.js]: Cool, there were a few more in other files as well.

Comment (talbet) [reply packages/components/src/DatePicker/components/CalendarDays/CalendarDays.js]: `CalendarNav.js`
`CalendarWeekdays.js`
`MaskedInput.js`

**Pull request merged**